README.md

```diff
@@ -4,13 +4,22 @@ Async Include
 
 A simple script to asynchronously include content on an HTML page with simple markup that “just works”.
 
-    <div data-async-url="/some-url/"></div>
+    <div data-async-include-url="/some-url/"></div>
+
+Just include the `async-include.js` file on your page, activate the replace when you want like when the document is ready:
+
+```javascript
+$(document).ready(function() {
+  $('[data-async-include-url]').asyncInclude();
+});
```

Comment:
actually in that 2nd code block, to be really flexible with when people can set `$.asyncInclude.noAutorun` it should probably not force people to wait for `$.asyncInclude` to exist (since they’d write something like `$.asyncInclude.noAutorun = true`), maybe just looking on the `window` value is an OK trade-off, e.g., allow people to do `window.ASYNC_INCLUDE_NO_AUTORUN = true;`